### PR TITLE
Added tables to make available options clearer

### LINKED ISSUE #184: Azure supports multiline secret values in bulk mode.
Azure support multiline secret values in bulk mode.

See:
- https://github.com/Consensys/web3signer/pull/827
- https://github.com/Consensys/web3signer/issues/780

### PR DESCRIPTION
**Pull request by MadelineMurray**
Also fixes #184

Pushing as a draft because this needs the GCP PR to be merged first so I can add the relevant links. 

@usmansaleem - is the point about 200 keys being able to be stored under a secret also true for Azure eth2 mode?  I copied it over from the AWS newline explanation.

### CODE REVIEW

**Comment by usmansaleem** [comment]
@MadelineMurray 👍🏼 Up to 200 keys per secret property is tested by us and it works both on AWS and Azure.

File: docs/how-to/load-keys.md [hunk 1]
```diff
@@ -8,19 +8,24 @@ import TabItem from '@theme/TabItem';
...
+
+| Key storage                          | Key configuration file | Bulk loading with `eth1` subcommand | Bulk loading with `eth2` subcommand | 
+|--------------------------------------|:----------------------:|:-----------------------------------:|:-----------------------------------:|
+| [Keystore files]                     |           x            |                  x                  |                  x                  | 
+| **Vaults**                           | 
+| [Hashicorp Vault]                    |           x            |                                     |                                     | 
+| [Azure Key Vault]                    |           x            |                  x                  |                  x                  | 
+| [AWS Secrets Manager]                |           x            |                                     |                  x                  | 
+| [AWS KMS]                            |           x            |                  x                  |                                     | 
+| [GCP Secret Manager]                 |           ?            |                                     |                  x                  |
```
Comment (usmansaleem) [inline docs/how-to/load-keys.md:22]: GCP is only supported via bulk loading and only on eth2 mode.

Comment (MadelineMurray) [reply docs/how-to/load-keys.md]: Have updated.

**Review by alexandratran — APPROVED** [review]
Minor edits, LGTM.

**Pull request merged**